go.mod

```diff
@@ -14,6 +14,9 @@ require (
 	github.com/atotto/clipboard v0.1.4 // indirect
```

Comment:
Can you explain why this file is modified? You should keep fuzzing module requirements to fuzzing targets only.